titus-common/src/test/java/com/netflix/titus/common/util/loadshedding/tokenbucket/ConfigurableTokenBucketAdmissionControllerTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.common.util.loadshedding.tokenbucket;
+
+import java.time.Duration;
+import java.util.List;
+
+import com.netflix.archaius.config.DefaultSettableConfig;
+import com.netflix.titus.common.framework.scheduler.model.ScheduleDescriptor;
+import com.netflix.titus.common.runtime.TitusRuntime;
+import com.netflix.titus.common.runtime.TitusRuntimes;
+import com.netflix.titus.common.util.Evaluators;
+import com.netflix.titus.common.util.loadshedding.AdmissionController;
+import com.netflix.titus.common.util.loadshedding.AdmissionControllerRequest;
+import com.netflix.titus.common.util.loadshedding.AdmissionControllerResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ConfigurableTokenBucketAdmissionControllerTest {
```

Comment:
add a test for invalid configurations (keep the previous last seen valid one)

Reply:
In `ArchaiusTokenBucketAdmissionConfigurationParser` we skip all invalid configurations (there are log entries for each failure).
This way a single bad configuration entry will not prevent us from loading other entries, for example on startup.